### PR TITLE
Moved DamlSubmission* into separate proto file [KVL-980]

### PR DESCRIPTION
**Pull request by miklos-da**
Summary of changes:
* Moved `DamlSubmission` and `DamlSubmissionBatch` into `wire_format.proto` to make it explicit that they are not persisted.

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/participant-state/kvutils/src/main/protobuf/com/daml/ledger/participant/state/kvutils/wire_format.proto [hunk 1]
```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```
Comment (SamirTalwar-DA) [inline ledger/participant-state/kvutils/src/main/protobuf/com/daml/ledger/participant/state/kvutils/wire_format.proto:1]: "Format" is pretty much implied by the fact that this is a Protocol Buffers file.

Just _wire.proto_ would be fine, IMO.

File: ledger/participant-state/kvutils/tools/engine-replay/src/replay/scala/ledger/participant/state/kvutils/tools/engine/replay/Replay.scala [hunk 2]
```diff
@@ -11,7 +11,7 @@ import com.daml.ledger.participant.state.kvutils.export.{
   ProtobufBasedLedgerDataImporter,
   SubmissionInfo,
 }
-import com.daml.ledger.participant.state.kvutils.{Envelope, Raw, DamlKvutils => Proto}
+import com.daml.ledger.participant.state.kvutils.{Envelope, Raw, WireFormat => Proto}
```
Comment (SamirTalwar-DA) [inline ledger/participant-state/kvutils/tools/engine-replay/src/replay/scala/ledger/participant/state/kvutils/tools/engine/replay/Replay.scala:14]: Just import it directly?

**Review by SamirTalwar-DA — APPROVED** [review]

File: ledger/participant-state/kvutils/src/main/protobuf/com/daml/ledger/participant/state/kvutils/wire_format.proto (hunk 1 above)
Comment (fabiotudone-da) [reply ledger/participant-state/kvutils/src/main/protobuf/com/daml/ledger/participant/state/kvutils/wire_format.proto:1]: Or `wire_kvutils.proto` if you want to keep the context in the name.

**Review by fabiotudone-da — APPROVED** [review]

File: ledger/participant-state/kvutils/src/main/protobuf/com/daml/ledger/participant/state/kvutils/wire_format.proto (hunk 1 above)
Comment (miklos-da) [reply ledger/participant-state/kvutils/src/main/protobuf/com/daml/ledger/participant/state/kvutils/wire_format.proto]: OK, I renamed it to `submission.proto` and put it under the package `kvutils.wire`, turning off generating an outer class by protoc.

**Pull request merged**